.github/workflows/pull_request.yml

```diff
@@ -27,3 +27,9 @@ jobs:
 
       - name: Build all configurations
         run: ./gradlew assemble
+
+      - name: Upload APK as Artifacts
+        uses: actions/upload-artifact@v2.2.0
+        with:
+          name: kiwix-android
+          path: app/build/outputs/apk/debug
```

Comment:
you can use wildcards no? Might have to check the upload-artifact docs. `**.apk`, that'll keep this future proof and ensure we get the custom app in here too

Reply:
Nicee, I didn't know about wildcards what they are and when to use 'em. Got it now, thanks!